src/components/SortByCategory.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+
+function SortByCategory (props) {
```

Comment:
the same thing for props and arrow component here

Reply:
Arrow Functionality Implemented